### PR TITLE
Assetrepo install

### PR DESCRIPTION
**Pull request by Kinshuk1993**
Assetrepo install
https://github.ibm.com/cp4i/icp4i-sagan/issues/661

### CODE REVIEW

File: products/bash/release-products.sh [hunk 1]
```diff
@@ -239,6 +240,56 @@ function release_apic {
...
+"
+
+function release_asset_repo {
+    echo "Releasing Asset Repo..."
+    # Validate the environment
+    if ! kubectl get ns ${asset_repo_namespace} > /dev/null 2>&1; then
+      echo "There is no namespace '${asset_repo_namespace}'" 1>&2
+      exit 1
+    else
+      echo "Namespace '${asset_repo_namespace}' exists"
```
Comment (rjd60) [inline products/bash/release-products.sh:277]: Is this for debugging? The other release functions don't do this, so either delete it here (as it's not hugely helpful) or make the others the same.

Likewise for some other echoes below.

File: products/bash/release-products.sh [hunk 2]
```diff
@@ -239,6 +240,56 @@ function release_apic {
...
+      exit 1
+    else
+      echo "The secret '${asset_repo_pull_secret}' exists in the namespace '${asset_repo_namespace}'"
+    fi
+    # Create the Helm values file
+    echo "${asset_repo_values}" > asset-repo-values.yaml
+    echo "Asset repository values.yaml created"
+    # Create the release
+    echo "Creating the asset repo release"
+    helm install ${chart_repo}/${asset_repo_chart} --name ${asset_repo_release_name} --namespace ${asset_repo_namespace} --values asset-repo-values.yaml --tls --debug
```
Comment (rjd60) [inline products/bash/release-products.sh:290]: And the `--debug` here should be removed.

File: products/bash/release-products.sh (hunk 1 above)
Comment (Kinshuk1993) [reply products/bash/release-products.sh]: Done

File: products/bash/release-products.sh (hunk 2 above)
Comment (Kinshuk1993) [reply products/bash/release-products.sh]: Done

**Pull request merged**